map.go

```diff
@@ -29,11 +30,23 @@ func markerHandler(c *gin.Context) {
 	markerID := c.Param("markerID")
 	fmt.Println("Marker ID: ", markerID)
 
+	// fake sensor types data with unit
+	sensorTypes := []map[string]string{
+		{"type": "temperature", "unit": "°C"},
+		{"type": "humidity", "unit": "%"},
+		{"type": "light", "unit": "lux"},
+		{"type": "moisture", "unit": "%"},
+	}
+
+	// current time
+	currentTime := time.Now().Format("2006-01-02 15:04:05")
```

Comment:
You can use the `time.DateTime` constant for this:
```suggestion
	currentTime := time.Now().Format(time.DateTime)
```